tagrecommendation/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "3.7"
+
+services:
+    tagrecommendation:
+        build: .
+        init: true
+        command: python tagrecommendation_server.py
+        volumes:
+            - .:/code
+            - ../freesound-data/tag_recommendation_models:/tag_recommendation_models
```

Comment:
looking at this again, I wonder if we should standardise on a `/data/` directory or similar in all images to store data. we can then have sub-directories named like this (`/tag_recommendation_models`)

Reply:
I was thinking about this. Yes we should, so all images share the same paths to data files as well. Eg: `/freesound-data/tag_recommendation_models/`, `/freesound-data/analysis/ ` ...

Reply:
This is also important because, for example, similarity has an endpoint to add a point to the gaia index and it takes as input the path to an analysis file. This path is sent from appserver. therefore appserver and similarity need to have the same path